index.js

```diff
@@ -187,6 +187,9 @@ function use (extensions) {
     this.on('close', onfinalize)
     this.on('end', onfinalize)
     this.on('finish', this.finalize)
+    this.on('error', function (err) {
+      debug('stream error', err)
+    })
```

Comment:
Will it? Duplexify uses a [standard error event](https://github.com/mafintosh/duplexify/blob/master/index.js#L184). Is there code somewhere that won't get triggered if we register a handler?